python/README.md

```diff
@@ -2,4 +2,13 @@ This folder contains the Python bindings and code required for embedding python
 
 For the Python bindings, see the [Python bindings of opm-simulators](https://github.com/OPM/opm-simulators/blob/master/python/README.md).
 
-For embedding python, see the [documentation](https://opm-project.org/?page_id=1454). 
\ No newline at end of file
+For embedding python, see the [documentation](https://opm-project.org/?page_id=1454).
+
+To enable tooltips for opm_embedded on VSCode (embedded python code): Copy the file "<opm-common-folder>/python/opm_embedded.pyi" in the folder at "python.analysis.stubPath" of VS Code
+You can recreate this stub file by executing "stubgen -m opmcommon_python" in "<opm-common-build-folder>/python/opm", renaming the resuling stub file from "opmcommon_python.pyi" to "opm_embedded.pyi" and adding the lines
```

Comment:
resuling -> resulting
Not 100% clear where these lines should be added (currently they are abov `class Builtin:`

So basically this could also be created on the fly if package mypy is installed using:
```
stubgen -m opmcommon_python -o .
cat opmcommon_python.pyi | sed "s/class Builtin:/current_ecl_state: EclipseState\ncurrent_report_step: int\ncurrent_schedule: Schedule\ncurrent_summary_state:\n\nclass Builtin:/" > opm_embedded.pyi
```